src/dao/pages/MultiSigDAODashboard/DepositTokensModal.tsx

```diff
@@ -34,28 +45,84 @@ export function DepositTokensModal(props: DepositTokensModalBodyProps) {
     <form onSubmit={handleSubmit(onSubmit)}>
       <Flex direction="column" gap={4}>
         <Divider marginX={-5} paddingX={5} />
-        <FormTokenInput
-          amountFormId="amount"
-          tokenFormId="tokenId"
-          assetListAccountId={safeId}
-          balanceAccountId={walletId ?? ""}
-          currentAmount={getValues().amount ?? ""}
-          form={form}
-          isInvalid={Boolean(errors?.amount)}
-          errorMessage={errors?.amount && errors?.amount?.message}
-        />
-        <Flex justify="space-around" gap={4}>
-          <Text textStyle="p xsmall regular">If you can't find the token you want to deposit, create a proposal</Text>
-          <Button
-            variant="secondary"
-            onClick={handleAssociateTokenClicked}
-            paddingX={4}
-            leftIcon={<NewTokenIcon boxSize="4" color={Color.Neutral._400} marginTop="0.2rem" />}
-            flexShrink={0}
-          >
-            Associate Token
-          </Button>
-        </Flex>
+        {tokens.filter((token) => token.isNFT).length > 0 && (
+          <Tabs isFitted onChange={handleTabChange} index={tabIndex} variant="dao-create-new" width="100%">
+            <DAOFormContainer rest={{ padding: "0.1rem" }}>
+              <TabList>
+                <Tab>Token</Tab>
+                <Tab>NFT</Tab>
+              </TabList>
+            </DAOFormContainer>
+          </Tabs>
+        )}
+        {tabIndex === 0 ? (
+          <>
+            <FormTokenInput
+              amountFormId="amount"
+              tokenFormId="tokenId"
+              assetListAccountId={safeId}
+              balanceAccountId={walletId ?? ""}
+              currentAmount={getValues().amount ?? ""}
```

Comment:
better to use destructured value `const { amount = "" } = getValues()` for all values used.

Reply:
Done.